GWAS/LMM.ipynb

```diff
@@ -1208,6 +1209,156 @@
     "1. A file with association results for each chromosome (Note: this are given in regard to Allele 2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "## GMMAT workflow implementation\n",
+    "Documentation can be found [here](https://github.com/hanchenphd/GMMAT/blob/master/inst/doc/GMMAT.pdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "### Step1: Creation of the GRM "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "#Calculate standardized GRM using GEMMA\n",
+    "[gemma_grm]\n",
+    "input:bfile\n",
+    "output:f'{cwd}/{bfile:bn}.sXX.txt'\n",
+    "task: trunk_workers = 1, trunk_size = job_size, walltime = '2h', mem = '6G', cores = 1, tags = f'{step_name}_{_output[0]:bn}'\n",
+    "bash: container=container_lmm, expand = \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout' \n",
+    "    gemma \\\n",
+    "    -bfile ${_input:n} \\\n",
+    "    -gk 2 \\\n",
+    "    -o ${_output:bnn}\n",
+    "bash: container=container_lmm,expand='${ }', stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout'\n",
+    "    cp ./output/${bfile:bn}.sXX.txt ${cwd}\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "### Step 2 :Fitting the null"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "# Run GMMAT step1: fit a GLMM with covariate adjustment and random effects to account for population structure and family or cryptic relatedness\n",
+    "[GMMAT_1]\n",
+    "#use the standardized GRM file generated in gemma steps\n",
+    "parameter: grmFile = path(f'{cwd}/{bfile:bn}.sXX.txt')\n",
+    "#a colum in the  data frame data, indicaing e id of samples\n",
+    "parameter: phenoCol = 'AD'\n",
+    "parameter: idCol = 'IID'\n",
+    "input: phenoFile, f'{bfile:n}.fam', grmFile\n",
+    "output: f'{cwd}/{bfile:bn}.{phenoFile:bn}.GMMAT.rds'\n",
+    "R: container=container_lmm,expand='${ }', stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout'\n",
+    "    library('GMMAT')\n",
+    "    library('data.table')\n",
+    "    library('dplyr')\n",
+    "   \n",
+    "    #Prepare phenotype and covariates in an R data frame\n",
+    "    pheno = data.table::fread(${_input[0]:r}, header = TRUE)\n",
```

Comment:
when you load `data.table` with `library` there is no need to use `data.table::`